public/app/plugins/datasource/tempo/SearchSettings.tsx

```diff
@@ -0,0 +1,37 @@
+import { css } from '@emotion/css';
+import { DataSourcePluginOptionsEditorProps, GrafanaTheme, updateDatasourcePluginJsonDataOption } from '@grafana/data';
+import { InlineField, InlineFieldRow, InlineSwitch, useStyles } from '@grafana/ui';
+import React from 'react';
+import { TempoJsonData } from './datasource';
+
+interface Props extends DataSourcePluginOptionsEditorProps<TempoJsonData> {}
+
+export function SearchSettings({ options, onOptionsChange }: Props) {
+  const styles = useStyles(getStyles);
+
+  return (
+    <div className={css({ width: '100%' })}>
```

Comment:
Why not move this to the `getStyles` as well? Also not sure why object syntax for styles is used here, I'd expect  `<div style={{ width: '100%' }}>` or ``<div className={css`width: 100% `}>``

Reply:
I'll clean that up